src/test/resources/readme_examples/he-service-a-gradle-readme.md

```diff
@@ -0,0 +1,113 @@
+# MicroProfile generated Application
+
+## Introduction
+
+MicroProfile Starter has generated this MicroProfile application for you.
+
+The generation of the executable jar file can be performed by issuing the following command
+
+
+
```

Comment:
Why do we need so many blank lines for the command? The same command applies to all the other command as well.

Reply:
Templating emgine leaves them there. I reduced their number in this pr.

The file you are commenting on is a static test for the 'parser'. Ignore it.